mtps-get-task

```diff
@@ -739,34 +711,44 @@ if [ -n "$SRPM" ]; then
     exit 0
 fi
 
-rpms_from_task_all="${rpms_from_task_all%%+( )}"
-rpms_from_task_noarch_all="${rpms_from_task_noarch_all%%+( )}"
 if [ -z "${rpms_from_task_all}${rpms_from_task_noarch_all}" ]; then
-    echo "Brew repo doesn't have rpms for $ARCH/noarch"
-    echo "Exit with special exit code: $RET_NO_PKG"
+    echo "There are no x86_64/noarch RPMs for Brew task $TASK"
```

Comment:
Somewhat is good, but we have ticket to extend mini-tps to other arches, then will necessary revert this